packages/renderer/src/stores/update-store.ts

```diff
@@ -0,0 +1,44 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Writable } from 'svelte/store';
+import { writable } from 'svelte/store';
+
+let podmanDesktopUpdateAvailable = false;
+
+try {
+  podmanDesktopUpdateAvailable = await window.podmanDesktopUpdateAvailable();
+} catch (e) {
+  console.log('Cannot check for update');
+}
+
+export const updateAvailable = setup();
+
+export function setup(): Writable<boolean> {
+  const store = writable(podmanDesktopUpdateAvailable);
+
+  window.events?.receive('app-update-available', isUpdateAvailable => {
+    if (isUpdateAvailable) {
+      updateAvailable.set(true);
+    } else {
+      updateAvailable.set(false);
+    }
+  });
+
+  return store;
+}
```

Comment:
if we go with a store, it should be like the other one and use the `new EventStore<....>` pattern (where you can add events that you're interested in)

you might look at existing stores importing `EventStore`